proto/commons/massa/model/v1/commons.proto

```diff
@@ -13,6 +13,17 @@ option php_namespace = "Com\\Massa\\Model\\V1";
 option ruby_package = "Com::Massa::Model::V1";
 option swift_prefix = "MMODEL";
 
+// Massa error
+message Error {
```

Comment:
We also have an Error message on ABI side.
Where not currently using any code field, do you ?
I'm asking to prevent clash when we will merge

Reply:
We could share the same structure, the errors codes will be written later

Reply:
Yes we probably should share this type (and possibly others).
We need to sync a bit more maybe merging common types in a dedicated pr then having our own pr relying on it
I'm gonna open a pr on what we have on the abi side for you to see

Reply:
The only issue is : `NativeAmount` which is not present in testnet 24